sections/05_treat.md

```diff
@@ -156,7 +156,7 @@ repositioning via deep learning is currently missing.
 
 #### Ligand-based prediction of bioactivity
 
-In the biomedical domain, high-throughput chemical screening aims to improve
+High-throughput chemical screening aims to improve
```

Comment:
I'd argue we need to keep `In the biomedical domain,`.  That may be implied by the review topic, but there is an entirely different line of high-throughput chemical screening in field such as materials science that has its own meaning.